pkgs/misc/drivers/sc-controller/default.nix

```diff
@@ -34,6 +35,7 @@ buildPythonApplication rec {
 
   preFixup = ''
     gappsWrapperArgs+=(--prefix LD_LIBRARY_PATH : "$LD_LIBRARY_PATH")
+    gappsWrapperArgs+=(--set GDK_PIXBUF_MODULE_FILE ${librsvg}/lib/gdk-pixbuf-2.0/2.10.0/loaders.cache)
```

Comment:
This should be the default. Maybe `strictDeps` are interfering like in https://github.com/NixOS/nixpkgs/issues/56943.

Reply:
I found this commit 6c5983a291530f040deb97881e80cca373a5b29e to be cause while bisecting, maybe it means something to you.

Reply:
Looks like you're right: settings `strictDeps` is working.